File: Vini´s projects/conta.js
let v1 = 10
let v2 = 20
let resultado = 0
let operador = '+'

if(operador == '+'){
    resultado = v1 + v2
    console.log("O resultado da soma é:", resultado)
}
else if(operador == '-'){
    resultado = v1 - v2
    console.log("O rsultado da subtração é:", resultado)
}
else if(operador == '*'){
    resultado = v1 * v2
    console.log("O resultado da multiplicação é:", resultado)
}
else if(operador == '/'){
    if(v2 === 0){
        console.log("Erro, divisão por zero")
    }
    else{
        resultado = v1 / v2
        console.log("O resultado da divisão é:", resultado)
    }
}

File: Aula facul/exercicos em C/conta.c
#include <stdio.h>

int main(){
    
    // variaveis
    float num1, num2, resultado;
    char operador;
    char pergunta = 'S';

    // repetição do codigo
    while (pergunta == 'S' || pergunta == 's')
    {
        printf("Informe o primeiro numero: ");
        scanf("%f", &num1);

        printf("Informe o segundo numero: ");
        scanf("%f", &num2);

        printf("Informe a operacao: ");
        scanf(" %c", &operador);

        // Contas
        if (operador == '+')
        {
            resultado = num1 + num2;
            printf("Resultado: %.2f\n", resultado);
        }

        else if (operador == '-')
        {
            resultado = num1 - num2;
            printf("Resultado: %.2f\n", resultado);
        }

        else if (operador == '*')
        {
            resultado = num1 * num2;
            printf("Resultado: %.2f\n", resultado);
        }

        else if (operador == '/')
        {
            if (num2 != 0)
            {
                resultado = num1 / num2;
                printf("Resultado: %.2f\n", resultado);
            }

            else
            {
                printf("Divisao por zero\n");
            }
        }

        else
        {
            printf("\nInforme um operador valido\n");
        }

        printf("\n Deseja continuar (S/N)?: ");
        scanf(" %c", &pergunta);
    }
    return 0;
}